src/aiai_eval/cli.py

```diff
@@ -76,6 +76,20 @@
     show_default=True,
     help="Whether extra input should be outputted during benchmarking",
 )
+@click.option(
+    "--track-carbon-emissions",
+    "-tce",
+    is_flag=True,
+    show_default=True,
+    help="Whether to track carbon usage.",
+)
+@click.option(
+    "--country-iso-code",
+    "-co",
+    default="DNK",
```

Comment:
I think this is potentially misleading, if they forget about this argument. Maybe set default to be an empty string, and raise an error if `--track-carbon-emissions` is set and this is not?

Reply:
Will do.